assets/javascripts/src/modules/giraffe.es6

```diff
@@ -128,60 +131,68 @@ function ophanId(){
     })
 }
 
-function carousel() {
-    const HIDDEN = 'js-hidden-tablet';
-    let $contribution = $('.js-contribution');
-    let $contributionButton = $('.js-advance',$contribution);
-    let $details = $('.js-details');
-    let $detailsButton = $('.js-advance', $details);
-    let $pay = $('.js-payment');
-    let all = Array.from($('.form__column'));
+function skipAmount() {
+    let skip = getSkipAmountFromQueryString( window.location.search );
```

Comment:
ah alright, I would probably rename the method shouldSkipAmount and make it return a boolean to make it clearer but that is just a detail.